application/src/main/resources/config/application-dev.yaml

```diff
@@ -49,5 +49,4 @@ keycloak:
   resource: ${keycloak.client-id}
   use-resource-role-mappings: true
 
-app:
-  hub-group-detail-base-url: http://hubdev.okd-entando.org/entando-de-app/en/test.page#/
+HUB_GROUP_DETAIL_BASE_URL: http://hubdev.okd-entando.org/entando-de-app/en/test.page#/
```

Comment:
@SACHIN-BAGHEL this is mixing the local stack settings with the integration environment setting.

Reply:
This one should probably be changed to http://localhost:3000/#/ so it's setup for a local environment,

Reply:
I Will make that change.

**application.yaml** - HUB_GROUP_DETAIL_BASE_URL: http://hubdev.okd-entando.org/entando-de-app/en/test.page#/
**application-dev.yaml** - HUB_GROUP_DETAIL_BASE_URL: http://localhost:3000/#/
**application-prod.yaml** - ?
 
**What is the expected value in application-prod.yaml ?**

Reply:
@SACHIN-BAGHEL , let's leave it commented out and empty in application.yaml and application-prod.yaml. Please also add a comment in the README about setting the value if you want active links back from the App Builder.